### PR TITLE
Add possibility to specify SSL files location via parameters.

### PR DESCRIPTION
**Pull request by rubenvp8510**
@josejulio Could you please review?

### CODE REVIEW

**Comment by rhqci** [comment]
Can one of the admins verify this patch?

File: docker-dist/src/main/docker/Dockerfile [hunk 1]
```diff
@@ -34,7 +34,10 @@ ENV HAWKULAR_BACKEND=cassandra \
     HAWKULAR_USER=jdoe \
     HAWKULAR_USE_SSL=false \
     HAWKULAR_DATA=/var/opt/hawkular \
-    HAWKULAR_METRICS_TTL=${metrics_ttl}
+    HAWKULAR_METRICS_TTL=${metrics_ttl}\
+    HAWKULAR_PUBLIC_KEY_FILENAME="/client-secrets/hawkular-services-public.pem" \
+    HAWKULAR_PRIVATE_KEY_FILENAME="/client-secrets/hawkular-services-public.pem" \
```
Comment (josejulio) [inline docker-dist/src/main/docker/Dockerfile]: This should be `/client-secrets/hawkular-services-private.key`

**Comment by jshaughn** [comment]
Just a reminder. The master branch is now for GA.  If this change is needed for TP3 (CF4.6 release) then a PR is needed for the 0.40.x branch.

**Comment by jshaughn** [comment]
@josejulio Just to confirm that this PR is for Services GA, not TP3.  If yes I can merge.

**Comment by josejulio** [comment]
@jshaughn I suppose is for both, maybe @theute knows about it.

**Pull request merged**